tests/components/freebox/conftest.py

```diff
@@ -65,25 +101,16 @@ def mock_device_registry_devices(hass: HomeAssistant, device_registry):
 @pytest.fixture(name="router")
 def mock_router(mock_device_registry_devices):
     """Mock a successful connection."""
+
     with patch("homeassistant.components.freebox.router.Freepybox") as service_mock:
-        instance = service_mock.return_value
-        instance.open = AsyncMock()
-        instance.system.get_config = AsyncMock(return_value=DATA_SYSTEM_GET_CONFIG)
-        # device_tracker
-        instance.lan.get_hosts_list = AsyncMock(return_value=DATA_LAN_GET_HOSTS_LIST)
-        # sensor
-        instance.call.get_calls_log = AsyncMock(return_value=DATA_CALL_GET_CALLS_LOG)
-        instance.storage.get_disks = AsyncMock(return_value=DATA_STORAGE_GET_DISKS)
-        instance.storage.get_raids = AsyncMock(return_value=DATA_STORAGE_GET_RAIDS)
-        instance.connection.get_status = AsyncMock(
-            return_value=DATA_CONNECTION_GET_STATUS
-        )
-        # switch
-        instance.wifi.get_global_config = AsyncMock(return_value=WIFI_GET_GLOBAL_CONFIG)
-        # home devices
-        instance.home.get_home_nodes = AsyncMock(return_value=DATA_HOME_GET_NODES)
-        instance.home.get_home_endpoint_value = AsyncMock(
-            return_value=DATA_HOME_PIR_GET_VALUES
-        )
-        instance.close = AsyncMock()
+        init_mock_router(service_mock, False)
+        yield service_mock
+
+
+@pytest.fixture(name="router_bridge_mode")
+def mock_router_bridge_mode(mock_device_registry_devices):
+    """Mock a successful connection to Freebox Bridge mode."""
+
+    with patch("homeassistant.components.freebox.router.Freepybox") as service_mock:
+        init_mock_router(service_mock, True)
```

Comment:
I think you can keep `mock_router` as it is and then:

```suggestion
@pytest.fixture(name="router_bridge_mode")
def mock_router_bridge_mode(mock_device_registry_devices, router):
    """Mock a successful connection to Freebox Bridge mode."""

    router().lan.get_hosts_list = AsyncMock(
        side_effect=HttpRequestError(
            "Request failed (APIResponse: %s)"
            % json.dumps(DATA_LAN_GET_HOSTS_LIST_MODE_BRIDGE)
        )
    )
```